test/consul.js

```diff
@@ -346,57 +346,6 @@ describe('Consul', () => {
     });
   });
 
-  it('resolves multiple tags as separate services', () => {
-    const consul = generateConsulStub();
-
-    consul.initialize();
-    consul.mock.emitChange('catalog-service', {consul: ['production', 'development']});
-    consul.mock.emitChange('consul-production', [{
-      Service: {Address: '127.0.0.1'}
-    }]);
-    consul.mock.emitChange('consul-development', [{
-      Service: {Address: '10.0.0.0'}
-    }]);
-
-    return Promise.resolve(consul.properties.consul).should.eventually.eql({
-      'consul-production': {
-        cluster: 'production',
-        addresses: ['127.0.0.1']
-      },
-      'consul-development': {
-        cluster: 'development',
-        addresses: ['10.0.0.0']
-      }
-    });
-  });
-
-  it('avoids resolving similar tags to the same service', () => {
```

Comment:
Is this test case (agents with different IP addresses and the same tag registering to different services should resolve the addresses to each individual service) covered elsewhere? If not, we might want to fix it so it passes instead of removing it.